extensions/wikia/Email/Controller/FounderController.class.php

```diff
@@ -318,7 +318,7 @@ private function getRecentActivity( $num = 5 ) {
 			'list' => 'recentchanges',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "recentchanges" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)